src/syrupy/assertion.py

```diff
@@ -295,8 +295,8 @@ def _assert(self, data: "SerializableData") -> bool:
         snapshot_name = self.extension.get_snapshot_name(
             test_location=self.test_location, index=self.index
         )
-        snapshot_data: Optional["SerializedData"] = None
-        serialized_data: Optional["SerializedData"] = None
+        snapshot_data: Optional[SerializedData] = None
```

Comment:
@epenet Does this require `from __future__ import annotations`? or this is fine in py3.8?

Reply:
It is fine as it is.
Fully compatible with 3.8